docker_builder/variables.tf

```diff
@@ -0,0 +1,21 @@
+variable "repository" {
+  type        = string
+  description = "URL of the ECR repository"
+}
+
+variable "service" {
+  type        = string
+  description = "Name of the service"
+}
```

Comment:
I think we can simplify this and directly ask for `repositoryURL` instead of asking for repo and service(also, service sounds bit confusing term here)

If one used , our ECR repo module, we already have added it as  [output](https://github.com/peak-ai/terraform-modules/blob/master/ecr/outputs.tf#L16-L19)

Reply:
Yeah good idea